packages/target-native-android-emulator/package.json

```diff
@@ -20,7 +20,7 @@
   ],
   "main": "src/index.js",
   "dependencies": {
-    "@ferocia-oss/osnap": "^1.3.0",
+    "@ferocia-oss/osnap": "^1.3.5",
```

Comment:
You also need to update the `yarn.lock` file. Run `yarn install` (or just `yarn`) in the root of the repo